tests/Service/Batch.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+use Resend\Collection;
+
+it('can send a  batch of emails', function () {
+    $client = mockClient('POST', 'emails/batch', [
+        [
+            'to' => 'test@resend.com',
+        ],
+        [
+            'to' => 'test@resend.com',
+        ],
+    ], batch());
+
+    $result = $client->batch->send([
+        [
+            'to' => 'test@resend.com',
+        ],
+        [
+            'to' => 'test@resend.com',
+        ],
```

Comment:
we only need `to` here? Or should we include the rest required ones?

Reply:
Honestly I should include all the required fields but the library relies on the API for validation and tests don't hit the actual API. I'll get that updated